forms-flow-api/src/formsflow_api/services/factory/keycloak_group_service.py

```diff
@@ -144,3 +144,20 @@ def format_response(self, data):
                 else ""
             )
         return data
+
+    def add_user_to_group_role(self, user_id: str, group_id: str, payload: Dict):
+        """Add user to group."""
+        data = {
+            "realm": current_app.config.get("KEYCLOAK_URL_REALM"),
+            "userId": payload.get("userId"),
+            "groupId": payload.get("groupId"),
+        }
+        return self.client.update_request(
+            url_path=f"users/{user_id}/groups/{group_id}", data=data
+        )
```

Comment:
Whether clientId, groupId in the payload & path param is the same?

Reply:
yes